client/src/components/SearchResult.js

```diff
@@ -49,10 +54,15 @@ export const SearchResult = ({ resource, children, hideDefaults = false }) => {
               <ResourceTypeIcon />
               {getReadable(resource.category)}
             </Box>
-            {resource.additional_metadata.organism && (
-              <Box as="span" gap="small" align="center" direction="row">
+            {resource.organism && (
+              <Box as="span" align="center" direction="row" gap="small">
                 <OrganismIcon />
-                {resource.additional_metadata.organism}
+                <span>
+                  {resource.organism.map((organism, i) => [
```

Comment:
Woah, `organism` is a list. What a twist!!! This should definitely be changed to `organisms`, but I can see that it's an issue even in the Django model.

Reply:
yeah we should rename it and when the organisms stop showing up we can switch it then